edgeless_embedded_emu/src/main.rs

```diff
@@ -30,7 +30,7 @@ async fn registration(agent: edgeless_embedded::agent::EmbeddedAgent) {
     let mut agent = agent;
     embassy_time::Timer::after_millis(5000).await;
     log::info!("Try Register!");
-    agent.register().await;
+    agent.register(Ipv4Address::new(192, 168, 101, 1)).await;
```

Comment:
Do you really need to have this hardcoded IP address? If yes, then fine. Otherwise, if you don't want to spend too much effort in creating a configuration file or command-line option etc, I suggest you consider reading from an environment variable that defaults to 192.168.101.1.

Reply:
OK, I feared something like that could be the case. Never mind my comment, then.

The bootstrapping of embedded devices is always painful, just look at how much work they have in (e.g.,) the OMA Device Management standardization group for that and similar amenities like over-the-air update...